lib/common.rb

```diff
@@ -0,0 +1,41 @@
+require 'openssl'
```

Comment:
The location of this file is problematic, as pluginsync will copy it to `$libdir/common.rb`.

This should go into some kind of `lib/puppet_x/openssl/` directory imo.

Reply:
what could possibly go wrong, as long there is no other file named common.rb ... 💥

Reply:
Have moved it, spec tests still green.